ui/lib/core/addon/components/search-select-placeholder.hbs

```diff
@@ -6,7 +6,7 @@
 <div>
   <div class="field">
     <p class="control has-icons-left has-icons-right">
-      <span class="input has-text-grey-light">{{or @placeholder "Search"}}</span>
+      <span class="input opacity-050">{{or @placeholder "Search"}}</span>
```

Comment:
Since this acts as a placeholder it should have the same visual style as a regular input. I noticed this when used side by side with `FilterInput` it looked like this:

![image](https://github.com/hashicorp/vault/assets/24611656/5b2c1f91-bee4-4ca9-8155-86d354aa9774)